PD-1924 / 25.10 / Pd 1924 update group privileges screen content (#3728)
truenas/documentation: content/SCALE/SCALEUIReference/Credentials/LocalGroupsScreens.md

## Privileges Screen

{{< hint type="warning" title="Experimental Feature" >}}

> The **Privileges** feature releases earlier than 24.10 is an experimental feature but as of 24.10 is no longer experimental.

Reviewer: ```suggestion
The **Privileges** feature in releases earlier than 24.10 is experimental, but as of 24.10 onwards is no longer experimental.
```

The new and edit privilege screens show the same settings but not all settings are editable.

{{< enterprise >}}

> Enterprise-licensed systems can enable Active Directory to provision groups in TrueNAS. To make this possible, go to the **System > Advanced Settings > Access** and enable the **Allow Directory Service users to access WebUI** option. After enabling this, the **Edit Privilege** screen lists AD groups on the **Groups** dropdown list.

Reviewer: ```suggestion
Enterprise-licensed systems can enable Active Directory to provision groups in TrueNAS. To make this possible, join Active Directory, then go to **System > Advanced Settings > Access** and enable the **Allow Directory Service users to access WebUI** option. After enabling this, the **Edit Privilege** screen lists AD groups on the **Groups** dropdown list.
```

> Enterprise-licensed systems can enable Active Directory to provision groups in TrueNAS. To make this possible, go to the **System > Advanced Settings > Access** and enable the **Allow Directory Service users to access WebUI** option. After enabling this, the **Edit Privilege** screen lists AD groups on the **Groups** dropdown list.

Reviewer: ```suggestion
Enterprise-licensed systems can enable Active Directory to provision groups in TrueNAS. To make this possible, join Active Directory, then go to **System > Advanced Settings > Access** and enable the **Allow Directory Service users to access WebUI** option. After enabling this, the **Edit Privilege** screen lists AD groups on the **Groups** dropdown list.
```

{{< truetable >}}
| Setting | Description |
|---------|-------------|
| **Name** | Assigns the name entered to a new privilege. Names can include the dash (-) or underscore(_) special characters, and upper and lowercase alphanumeric characters. Enter a descriptive name for the privilege. **Name** shows on the **Edit Privilege** screen but is not editable. |
| **Groups** | Shows a list of groups configured on the system. Select a group from the dropdown list after clicking in the field. The privilege is applied to the selected group(s). |

> | **Roles** |Select from a dropdown list of all available roles available to assign to the new privilege, or change in an existing privilege. Note, only Readonly Admin, Sharing Admin, or Full Admin roles are supported in the web UI. |

Reviewer: ```suggestion
| **Roles** | Select from a dropdown list of all available roles available to assign to the new privilege or change an existing privilege. Only the Readonly Admin, Sharing Admin, or Full Admin roles are supported in the web UI. |
```
